### PR TITLE
mention ticket price and info@

### PR DESCRIPTION
**Pull request by zormit**

### CODE REVIEW

**Comment by hellerve** [comment]
Looks good!

Maybe rephrase the last sentence:

```
For the talks we have space for up to 100 people and the attendance will be free
(although we might charge a no-show-fee).

// =>

We have room for up to 100 people at the venue and attendance will be free,
although we might charge a no-show fee.
```

**Comment by Dzol** [comment]
I made the suggestions here coz zormit is busy. Let me know you thoughts @hellerve if any. I'll aim to merge the PRs in a few hours or this evening

**Comment by hellerve** [comment]
:+1: LGTM
